service/deployer/eventer/eventer.go

```diff
@@ -27,73 +35,42 @@ func DefaultConfig() Config {
 		Logger: nil,
 
 		// Settings.
-		Type: StubEventer,
+		Type: github.GithubEventer,
 	}
 }
 
 // New creates a new configured Eventer.
-func New(config Config) (Eventer, error) {
+func New(config Config) (spec.Eventer, error) {
```

Comment:
Actually, gonna disagree. In the future, we might want to return different eventer types.

I would agree that the `New` in `github.go` makes sense to return a pointer to the `githubEventer` struct.